overlord/snapstate/refresh.go

```diff
@@ -20,20 +20,19 @@
 package snapstate
 
 import (
-	"bufio"
 	"fmt"
-	"io"
-	"os"
-	"path/filepath"
 	"sort"
-	"strconv"
 	"strings"
 
 	"github.com/snapcore/snapd/cmd/snaplock"
-	"github.com/snapcore/snapd/dirs"
+	"github.com/snapcore/snapd/sandbox/cgroup"
 	"github.com/snapcore/snapd/snap"
 )
 
+var (
+	pidsCgroupDir = cgroup.ControllerPathV1("pids")
```

Comment:
Nice

Once the name=snapd cgroup lands this will change to just `/run/snapd/cgroup` right?

Reply:
Yes